picard/util/tags.py

```diff
@@ -165,653 +343,1119 @@ def names(self, selector=None):
+        is_from_mb=False,
     ),
     TagVar(
         'filename',
-        shortdesc=N_('File name'),
+        shortdesc=N_('File Name'),
+        longdesc=N_('The name of the file without extension.'),
         is_hidden=True,
         is_preserved=True,
         is_tag=False,
+        is_from_mb=False,
     ),
     TagVar(
         'filepath',
         shortdesc=N_('File Path'),
+        longdesc=N_('Full path and name of the file.'),
         is_hidden=True,
         is_tag=False,
+        is_from_mb=False,
     ),
     TagVar(
         'filesize',
-        shortdesc=N_('File size'),
+        shortdesc=N_('File Size'),
+        longdesc=N_('Size of the file in bytes.'),
         is_file_info=True,
         is_hidden=True,
         is_tag=False,
+        is_from_mb=False,
     ),
     TagVar(
         'format',
-        shortdesc=N_('File format'),
+        shortdesc=N_('File Format'),
+        longdesc=N_('Media format of the file (e.g.: MPEG-1 Audio).'),
         is_file_info=True,
         is_hidden=True,
         is_preserved=True,
         is_tag=False,
+        is_from_mb=False,
     ),
     TagVar(
         'gapless',
         shortdesc=N_('Gapless Playback'),
+        longdesc=N_('Indicates whether or not there are gaps between the recordings on the release.'),
+        is_from_mb=False,
     ),
     TagVar(
         'genre',
         shortdesc=N_('Genre'),
+        longdesc=N_(
+            'A multi-value tag containing the specified genre information from MusicBrainz.'
+        ),
+        related_options=('use_genres', ),
     ),
     TagVar(
+        # TODO: Check if this actually exists or if it was provided by the last.fm plugin.
         'grouping',
         shortdesc=N_('Grouping'),
+        longdesc=N_('Genre grouping associated with the track'),
+        is_from_mb=False,
     ),
     TagVar(
         'isrc',
         shortdesc=N_('ISRC'),
+        longdesc=N_(
+            'The International Standard Recording Code - an international standard code for uniquely '
+            'identifying sound recordings and music video recordings.'
+        ),
     ),
     TagVar(
         'key',
         shortdesc=N_('Key'),
+        longdesc=N_('The key of the music.'),
+        is_from_mb=False,
     ),
     TagVar(
         'label',
         shortdesc=N_('Record Label'),
+        longdesc=N_('A multi-value tag containing the names of the labels associated with the release.'),
     ),
     TagVar(
         'language',
         shortdesc=N_('Language'),
+        longdesc=N_('Work lyric language as per ISO 639-3 if a related work exists.'),
     ),
     TagVar(
         'length',
         shortdesc=N_('Length'),
+        longdesc=N_('The length of the track in format mins:secs.'),
         is_hidden=True,
         is_tag=False,
     ),
     TagVar(
         'license',
         shortdesc=N_('License'),
+        longdesc=N_('The licenses associated with the track, either through the release or recording relationships.'),
```

Comment:
Just to be sure `licenses` with a `s` is intentional here?

Reply:
Not intentional.  I just copied what was written in the docs.  It probably makes more sense shown as singular.